src/sage/schemes/elliptic_curves/ell_point.py

```diff
@@ -3950,20 +3955,31 @@ def log(self, base):
         For anomalous curves with `\#E = p`, the
         :meth:`padic_elliptic_logarithm` function is called.
 
+        For two-dimensional logarithms, we first compute the Weil pairing of
+        `Q` with `P_1` and `P_2` and their logarithms to base the pairing of
+        `P_1` and `P_2`; this allows to read off `x` and `y` modulo the part
+        of the order where `P_1` and `P_2`. Modulo the rest of the order the
```

Comment:
not sure i understand this comment --

> modulo the part of the order where `P_1` and `P_2`

Reply:
should this be "modulo the part of the order where P_1 and P_2 are independent"?

Reply:
Indeed, that was a typo, thanks. Fixed.